examples/ex18p.cpp

```diff
@@ -1,24 +1,29 @@
-//                       MFEM Example 18 - Parallel Version
+//                                MFEM Example 18 - Parallel Version
 //
-// Compile with: make ex18
+// Compile with: make ex18p
 //
 // Sample runs:
 //
-//       mpirun -np 4 ex18p -p 1 -rs 2 -rp 1 -o 1 -s 3
-//       mpirun -np 4 ex18p -p 1 -rs 1 -rp 1 -o 3 -s 4
-//       mpirun -np 4 ex18p -p 1 -rs 1 -rp 1 -o 5 -s 6
-//       mpirun -np 4 ex18p -p 2 -rs 1 -rp 1 -o 1 -s 3
-//       mpirun -np 4 ex18p -p 2 -rs 1 -rp 1 -o 3 -s 3
+//       mpirun ex18p -p 1 -rs 2 -o 1 -s 3
+//       mpirun ex18p -p 1 -rs 1 -o 3 -s 4
+//       mpirun ex18p -p 1 -rs 0 -o 5 -s 6
+//       mpirun ex18p -p 2 -rs 1 -o 1 -s 3
+//       mpirun ex18p -p 2 -rs 0 -o 3 -s 3
```

Comment:
Should these changes the sample runs be reverted? At least the `-np 4` should be brought back.